app/js/user.js

```diff
@@ -120,13 +133,15 @@
      * @returns The updated user object from indexedDB
      */
     update_user: function(user_data) {
-      return SumoDB.update_user(user_data).then(function(user) {
-        var normalized_user = normalize_user(user);
-        normalized_user.last_sync = Date.now();
-        return asyncStorage.setItem('user', normalized_user).then(function() {
-          return normalized_user;
-        });
-      });
+      return SumoDB.update_user(user_data).then(sync_user);
+    },
+
+    authenticate_user: function(username, password) {
+      return SumoDB.get_token(username, password);
+    },
+
+    request_password_reset: function(username) {
```

Comment:
No need to create this proxy function. We're not dealing with user data so we can call SumoDB directly.